Candidates.txt

```diff
@@ -0,0 +1,2 @@
+Jovan Stosic jovan-stosic
```

Comment:
If you started with fresh directory you shouldn't have this record - Jovan Stosic
Where did you get it. If you think that you don't have a time to start with fresh new directory and do all exercises again, then remove that record.